app/connectors/DesConnector.scala

```diff
@@ -163,6 +165,13 @@ class DesConnectorImpl @Inject()(
   }
 
   private def validateJson(json: JsValue): PsaSubscription ={
+
+    if (json.transform(psaSubscriptionDetailsTransformer.transformToUserAnswers).isSuccess)
+      Logger.warn("PensionAdministratorSuccessfulMapToUserAnswers")
```

Comment:
Should a successful response yield a WARN logging message? Info or debug would be better don't you think? Warn implies there's a potential problem.

Reply:
changed to info as suggested